includes/admin/class-sensei-learner-management.php

```diff
@@ -123,7 +122,7 @@ public function add_custom_navigation() {
 		if ( ! $screen ) {
 			return;
 		}
-
+		
```

Comment:
I think the empty space here [makes the linter angry](https://github.com/Automattic/sensei/runs/7303818585?check_suite_focus=true#step:8:18). 😄 Could you remove it?